Carbon.IIS/Functions/Disable-CIisSecurityAuthentication.ps1

```diff
@@ -20,23 +21,27 @@ function Disable-CIisSecurityAuthentication
     Test-CIisSecurityAuthentication
 
     .EXAMPLE
-    Disable-CIisSecurityAuthentication -SiteName Peanuts -Anonymous
+    Disable-CIisSecurityAuthentication -LocationPath 'Peanuts' -Anonymous

```

Comment:
`-SitePath` ?

Reply:
No, because it can operate on a website or any directory under a website. In Microsoft's docs, these are called locations, so I use `LocationPath`.